src/control/cmd/dmg/pretty/pool_test.go

```diff
@@ -1197,9 +1203,31 @@ no pools in system
 			},
 			verbose: true,
 			expPrintStr: `
-Label UUID                                 State SvcReps SCM Size SCM Used SCM Imbalance NVME Size NVME Used NVME Imbalance Disabled UpgradeNeeded? 
------ ----                                 ----- ------- -------- -------- ------------- --------- --------- -------------- -------- -------------- 
--     00000001-0001-0001-0001-000000000001 Ready N/A     100 GB   80 GB    12%           6.0 TB    5.0 TB    1%             0/16     1->2           
+Label UUID                                 State SvcReps SCM Size SCM Used SCM Imbalance NVME Size NVME Used NVME Imbalance Disabled UpgradeNeeded? Rebuild State 
+----- ----                                 ----- ------- -------- -------- ------------- --------- --------- -------------- -------- -------------- ------------- 
+-     00000001-0001-0001-0001-000000000001 Ready N/A     100 GB   80 GB    12%           6.0 TB    5.0 TB    1%             0/16     1->2                         
+
+`,
+		},
+		"verbose; zero svc replicas with no query": {
+			resp: &control.ListPoolsResp{
+				Pools: []*control.Pool{
+					{
+						UUID:             test.MockUUID(1),
+						Usage:            exampleUsage,
+						TargetsTotal:     16,
+						TargetsDisabled:  0,
+						State:            system.PoolServiceStateReady.String(),
+						PoolLayoutVer:    1,
+						UpgradeLayoutVer: 2,
+					},
+				},
+			},
+			verbose: true,
+			expPrintStr: `
```

Comment:
noQuery is not actually set to true for this test case. Based on the code I would expect the final column to not have been included.